### ISSUE TITLE
Failure if "disk_size" given as int

### ISSUE THREAD

**Issue by nhammond** [StanfordBioinformatics/loom#326]
I produced this error by modifying the first step of "hello_world" to give "disk_size" using an int rather than a string. This error is in the docker logs of the worker container.
```
[2017-03-22 20:11:18,035: WARNING/MainProcess] Exception in thread Thread-82:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/loomengine/loomengine/master/api/tasks.py", line 169, in _run_task_runner_playbook
    env.update(new_vars)
  File "/usr/lib/python2.7/os.py", line 488, in update
    self[k] = dict[k]
  File "/usr/lib/python2.7/os.py", line 473, in __setitem__
    putenv(key, item)
TypeError: must be string, not int
```

**Comment by nhammond** [comment]
Tested in release 0.5.2, using JSON template with int value in "disk_size" field. This works now.

**Comment by nhammond** [comment]
Fixed in 78c998cb628f46a8dda90c09cf52832b56877a9b
